langchain/src/agents/agent_toolkits/index.ts

```diff
@@ -13,3 +13,4 @@ export {
   createVectorStoreRouterAgent,
 } from "./vectorstore/vectorstore.js";
 export { ZapierToolKit } from "./zapier/zapier.js";
+export { AWSSfnToolkit, createAWSSfnAgent } from "./aws_sfn/aws_sfn.js";
```

Comment:
Can't export this from the main entrypoint since it requires a third-party dep - will break some environments:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#creating-a-separate-entrypoint